src/common/types/value.cpp

```diff
@@ -207,7 +215,7 @@ void Value::copyValueFrom(const uint8_t* value) {
     } break;
     case LogicalTypeID::MAP:
     case LogicalTypeID::VAR_LIST: {
-        copyFromVarList(*(ku_list_t*)value, *VarListType::getChildType(&dataType));
+        copyFromVarList(*(ku_list_t*)value, *VarListType::getChildType(dataType.get()));
```

Comment:
I think we should pass reference instead of raw pointer in this API as well? Let's open an issue for this.